script/update_eligibility.rb

```diff
@@ -76,19 +76,19 @@ def check_and_run
       person = Person.by_ssn(ssn).first rescue nil
     end
 
-    person_by_hbx = Person.by_hbx_id(hbx_id).first rescue nil
+    person_by_hbx = Person.by_hbx_id(person_hbx_id).first rescue nil
```

Comment:
Should be primary person hbx_id. Since we should be finding family using primary person hbx_id.